src/main/java/com/epam/repository/PersonDAO.java

```diff
@@ -32,4 +32,52 @@ Person buildPerson(ResultSet rs) throws SQLException {
                 .build();
     }
 
+    public boolean doesEmailExists(String email) {
+        List<String> emailsList = jdbcTemplate.query("SELECT email FROM person", (rs, rowNum) -> getStringEmail(rs));
+        for(String stringEmail:  emailsList){
+            if (stringEmail.equals(email)) return true;
+        }
+        return false;
+    }
+
+    String getStringEmail(ResultSet rs) throws SQLException {
```

Comment:
I think method name could be just `getEmail`

Reply:
Done